driver/src/driver/stablex_driver.rs

```diff
@@ -4,7 +4,7 @@ use crate::price_finding::PriceFinding;
 
 use dfusion_core::models::Solution;
 
-use log::info;
+use log::{debug, info};
```

Comment:
This use statement is not required - linting is failing the build because of it.